packages/comms/package.json

```diff
@@ -59,30 +54,30 @@
         "wsdl-topology": "node ./lib-cjs/index.js -k --url=http://localhost:8010/WsTopology?wsdl --outDir=./src/services/wsdl",
         "wsdl-workunits": "node ./lib-cjs/index.js -k --url=http://localhost:8010/WsWorkunits?wsdl --outDir=./src/services/wsdl",
         "wsdl-all": "npm-run-all --aggregate-output -c --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-esdlconfig wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-machine wsdl-packageprocess wsdl-smc wsdl-resources wsdl-store wsdl-topology wsdl-workunits",
-        "wsdl": "npm-run-all --aggregate-output -c --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-packageprocess wsdl-smc wsdl-store wsdl-topology wsdl-workunits"
+        "wsdl": "npm-run-all --aggregate-output -c --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-packageprocess wsdl-smc wsdl-store wsdl-topology wsdl-workunits",
```

Comment:
I think I noticed this yesterday and forgot to comment, but are "wsdl" and "wsdl-all" the same thing? Old merge conflict or something?

Reply:
There not - going to keep the longer one (wsdl-all)